qemu/tests/cfg/numa_opts.cfg

```diff
@@ -72,20 +72,32 @@
             policy_mem = default
             numa_cmd = "cat /sys/devices/system/node/possible"
             numa_expected = "0-7"
+        - nodes.64:
+            only aarch64
```

Comment:
Only available for aarch64?

Reply:
Other platforms Numa max nodes covered by nodes.128